components/atomic/RadialMeter/index.js

```diff
@@ -47,31 +44,38 @@ const RadialMeter = ({ min, max, value, text, step = 1, labelledById }) => {
           fill="none"
           strokeWidth="4"
           transform-origin={origin}
-          transform={`rotate(${value * degreeStep + baseRotation})`}
+          style={{
+            transform: `rotate(${value * degreeStep + baseRotation}deg)`,
+          }}
         />
         <g fill="#c3c3c3" strokeWidth="2">
           <circle cx={centerX} cy={centerY} r="4" stroke="none" />
           <circle cx={centerX} cy={centerY} r="3" fill="none" />
         </g>
       </g>
-      <g strokeWidth="2" stroke="#F5F5F5" strokeDasharray="6 24">
+      <g stroke="#F5F5F5">
         {range.map((v, i) => (
-          <line
-            key={v}
-            x1={centerX}
-            x2={centerX}
-            y1={centerY - 30}
-            y2={centerY}
+          <g
             transform-origin={origin}
-            transform={`rotate(${i * degreeStep + baseRotation})`}
-          />
+            style={{
+              transform: `rotate(${i * degreeStep + baseRotation}deg)`,
+            }}
+            key={v}
+          >
```

Comment:
instead of moving the text along a path, I am attaching the text to it's corresponding tick and offsetting then rotating it from the center